modules/openapi-generator/src/main/resources/aspnetcore/2.0/model.mustache

```diff
@@ -92,6 +92,7 @@ namespace {{packageName}}.Models
                 (
                     {{name}} == other.{{name}} ||
                     {{name}} != null &&
+                    other.{{name}} != null &&
```

Comment:
Good catch. `SequenceEqual` method delegates to the static [Enumerable.SequenceEqual](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=netframework-4.8#System_Linq_Enumerable_SequenceEqual__1_System_Collections_Generic_IEnumerable___0__System_Collections_Generic_IEnumerable___0__System_Collections_Generic_IEqualityComparer___0__) which will throw an argument null exception when this input is null.